File: C++/2914. Minimum Number of Changes to Make Binary String Beautiful.cpp
class Solution {
public:
    int minChanges(string s) {
        int ans=0;
        for(int i=0;i<s.size()-1;i+=2)
        {
            if(s[i]!=s[i+1])
            {
                ans++;
            }

        }
        if(s.size()%2)
        ans++;
        return ans;
    }
};

File: JAVA/2914. Minimum Number of Changes to Make Binary String Beautiful.java
class Solution {
    public int minChanges(String s) {
        char[] arr = s.toCharArray();
        int cnt = 0;
        for (int i = 0; i < arr.length - 1; i += 2) {
            if (arr[i] != arr[i + 1]) {
                cnt++;
            }
        }        
        return cnt;
    }
}
